modules/javafx.graphics/src/main/java/com/sun/javafx/font/PrismFontFile.java

```diff
@@ -114,37 +112,32 @@ void setIsDecoded(boolean decoded) {
 
     /* This is called only for fonts where a temp file was created
      */
-    @SuppressWarnings("removal")
     protected synchronized void disposeOnShutdown() {
         if (isCopy || isDecoded) {
-            AccessController.doPrivileged(
-                    (PrivilegedAction<Void>) () -> {
-                        try {
-                            /* Although there is likely no harm in calling
-                             * delete on a file > once, we want to refrain
-                             * from deleting it until the shutdown hook
-                             * code in subclasses has had an opportunity
-                             * to clean up native accesses on the resource.
-                             */
-                            if (decFileRefCount() > 0) {
-                                return null;
-                            }
-                            boolean delOK = (new File(filename)).delete();
-                            if (!delOK && PrismFontFactory.debugFonts) {
-                                 System.err.println("Temp file not deleted : "
-                                                    + filename);
-                            }
-                            /* Embedded fonts (copy) can also be decoded.
-                             * Set both flags to false to avoid double deletes.
-                             */
-                            isCopy = isDecoded = false;
-                        } catch (Exception e) {
-                        }
-                        return null;
-                    }
-            );
-            if (PrismFontFactory.debugFonts) {
-                System.err.println("Temp file deleted: " + filename);
+            try {
+                /* Although there is likely no harm in calling
+                 * delete on a file > once, we want to refrain
+                 * from deleting it until the shutdown hook
+                 * code in subclasses has had an opportunity
+                 * to clean up native accesses on the resource.
+                 */
+                if (decFileRefCount() > 0) {
+                    return;
+                }
+                boolean delOK = (new File(filename)).delete();
+                if (!delOK && PrismFontFactory.debugFonts) {
+                        System.err.println("Temp file not deleted : "
+                                        + filename);
+                }
+                /* Embedded fonts (copy) can also be decoded.
+                    * Set both flags to false to avoid double deletes.
+                    */
+                isCopy = isDecoded = false;
+            } catch (Exception e) {
+            } finally {
```

Comment:
even though the code change is equivalent, it will print "temp file not deleted" followed by "temp file deleted".

Reply:
well, the original code is just sloppy, but we have a policy of not fixing unrelated issues, or even adding comments pointing out the sloppy code.

in this case, we can probably ignore it - it's probably not worth fixing since it's a debug output.

Reply:
In this case, I wouldn't mind adding a comment about further follow-up, although I question whether it is worth it. Probably not (so you could just leave it alone).

Reply:
> I have a separate question: Is the finally statement really needed?

Not really, I think I just repeated the pattern from other similar blocks. I'll remove it.